src/main/java/org/bytedeco/javacpp/Loader.java

```diff
@@ -1357,6 +1328,93 @@ public static String load(Class cls, Properties properties, boolean pathsFirst)
         }
     }
 
+    /** Returns {@link Loader#loadExecutable(Class, String, Properties, boolean)} default values*/
+    public static String loadExecutable(Class cls, String name) {
+        return loadExecutable(cls, name, loadProperties(), Loader.pathsFirst);
+    }
+    /**
+     * Loads the path to an executable associated with a {@link Class} by its name
+     *
+     * This invokes a call to {@link Loader#load(Class, Properties, boolean)} which means the
+     * class will be loaded when when its executables are requested.
+     *
+     * @param cls the Class to load and locate the associated executable for
+     * @param name the executable program name, without prefix. (Eg: clang, not clang.exe)
+     * @param properties the platform Properties to inherit
+     * @param pathsFirst search the paths first before bundled resources
+     * @return the full path to the executable file found, or null if no executable with the given
+     *         name was found
+     * @see #load(Class, Properties, boolean)
+     */
+    public static String loadExecutable(Class cls, String name, Properties properties, boolean pathsFirst) {
+        // Load the library in case it hadn't been loaded previously
```

Comment:
I'm not sure how big of an overhead running listExecutables twice (because it's called through load()) is. Something to consider.